applications/launchpad_v2/src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react'
+
+import Footer from '.'
+
+describe('Footer', () => {
+  it('should render without crash', () => {
+    render(<Footer />)
+
+    const el = screen.getByTestId('footer-cmp')
```

Comment:
Not sure what do you mean by this...

Reply:
the footer component is so simple that this test seems unnecessary and it looks like it tests the library if it can find test-id etc etc
it's me complaining about minor stuff, ignore it ^^